TundraTD/Assets/Scripts/City/Building/TowerPlacementSlot.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Level;
+using UnityEngine;
+
+namespace City.Building
+{
+    /// <summary>
+    /// A slot to build a new tower 
+    /// </summary>
+    public class TowerPlacementSlot : MonoBehaviour
+    {
+        [SerializeField] private TowerPurchaseMenu purchaseMenu;
+        [SerializeField] private int slotID;
+        
+        private Vector3 _bottomCentreBuildingAnchor;
+        public bool IsOccupied { get; private set; }
+        public int SlotID => slotID;
+
+        public void BuildElementalTowerOnThisSlot(ElementalTower prefab)
```

Comment:
Nit: I think there was a good idea to make documentation for methods with long title

Reply:
I use long names to prevent writing incode documentation